docs/source/beta.rst

```diff
@@ -1,74 +1,152 @@
-Beta User Guide
-===============
+=====================
+Quipucords User Guide
+=====================
+Sonar Project, Pilot Phase
```

Comment:
We are externally using the term "Sonar Project"?

Reply:
For this comment specifically. Well...Dan is in his beta/pilot docs like the invitation and (I think) the slides. But I can have an additional conversation with him about vocabulary that is going to get exposed.

Reply:
RE the merge conflicts--more likely to happen with any doc files that are in the code, because they are worked on for (probably) longer chunks of time before merging than code files. If there's an adjustment to my workflow that would help because of the nature of that (such as any changes to steps I do BEFORE the push and PR creation), let's take another look at the workflow doc I sent you and see if we can improve the steps.

Reply:
RE the merge to master: No, my intent was to merge to the beta branch, as far as I know. Just a duh moment where I probably should have looked harder at the screen and realized I need to change what I was doing.

Reply:
I think we're good using "Project Sonar" externally.  My vocabulary has stabilized somewhat, and:

* Our collection of projects is the Subscription Reporting Initiative
* Our Project to find and "meter" Infrastructure is Project Sonar
* Our specific tool is quipucords or qpc

In this way, Sonar is reduced to a project name - which can be anything - and we set this repo up to be the community upstream for any potential products that we might become.

Reply:
I created a new issue to cover the final edit for the pilot launch. It includes the "known to-dos" mentioned in the creation of this PR. I also added another to-do to check for any mentions of Sonar in the doc and change to "Project Sonar" (I confirmed via Slack that Dan prefers this over "Sonar Project").
See issue #807

Reply:
Now that "project" is approved as okay for Sonar and I have recorded a to-do global check for all "Sonar" usage in #807, is it cool to use the Dismiss Review function? (Unsure if it will work anyway if that takes me down to no reviewers.) Thanks.